README.md

```diff
@@ -1,344 +1,42 @@
-4. Optionally, you can create a `.tfswitch.toml` file in your terraform directory(current directory) OR in your home directory(~/.tfswitch.toml). The toml file in the current directory has a higher precedence than toml file in the home directory
-5. Your `.tfswitch.toml` file should look like this:
-```ruby
-bin = "/Users/warrenveerasingam/bin/terraform"
-version = "0.11.3"
-```
-4. Run `tfswitch` and it should automatically install the required terraform version in the specified binary path
-
-**NOTE** 
-1. For linux users that do not have write permission to `/usr/local/bin/`, `tfswitch` will attempt to install terraform at `$HOME/bin`. Run `export PATH=$PATH:$HOME/bin` to append bin to PATH  
-2. For windows host, `tfswitch` need to be run under `Administrator` mode, and `$HOME/.tfswitch.toml` with `bin` must be defined (with a valid path) as minimum, below is an example for `$HOME/.tfswitch.toml` on windows
-
-```toml
-bin = "C:\\Users\\<%USRNAME%>\\bin\\terraform.exe"
-```
-### Use .tfswitchrc file
-<img src="https://s3.us-east-2.amazonaws.com/kepler-images/warrensbox/tfswitch/tfswitch-v6.gif#1" alt="drawing" style="width: 370px;"/>
-
-1. Create a `.tfswitchrc` file containing the desired version
-2. For example, `echo "0.10.5" >> .tfswitchrc` for version 0.10.5 of terraform
-3. Run the command `tfswitch` in the same directory as your `.tfswitchrc`
-
-#### *Instead of a `.tfswitchrc` file, a `.terraform-version` file may be used for compatibility with [`tfenv`](https://github.com/tfutils/tfenv#terraform-version-file) and other tools which use it*
-
-### Use terragrunt.hcl file
-If a terragrunt.hcl file with the terraform constrain is included in the current directory, it should automatically download or switch to that terraform version. For example, the following should automatically switch terraform to the latest version 0.13:
-```ruby
-terragrunt_version_constraint = ">= 0.26, < 0.27"
-terraform_version_constraint  = ">= 0.13, < 0.14"
-...
-```
-
-### Get the version from a subdirectory
-```bash
-tfswitch --chdir terraform_dir
-tfswitch -c terraform_dir
-```
-
-### Use custom mirror 
-To install from a remote mirror other than the default(https://releases.hashicorp.com/terraform). Use the `-m` or `--mirror` parameter.    
-Ex: `tfswitch --mirror https://example.jfrog.io/artifactory/hashicorp`
-
-### Set a default TF version for CICD pipeline
-1. When using a CICD pipeline, you may want a default or fallback version to avoid the pipeline from hanging.
-2. Ex: `tfswitch -d 1.2.3` or `tfswitch --default 1.2.3` installs version `1.2.3` when no other versions could be detected.
-[Also, see CICD example](#cicd)
-
-## Automation
-**Automatically switch with bash**
-
-Add the following to the end of your `~/.bashrc` file:
-(Use either `.tfswitchrc` or `.tfswitch.toml` or `.terraform-version`)
+## Documentation
+Click [here](https://tfswitch.warrensbox.com) for our extended documentation.
```

Comment:
I don't think we should entirely remove the installation instructions! 

Maybe we can have a subheading for Installation with a link to the relevant install page here instead, including listing all the ways installation is supported? (eg. via Brew, build from source, go get etc)

Reply:
+1